util/object.go

```diff
@@ -180,17 +180,21 @@ func NewClusterIPService(ns, name string, port, targetPort int) *corev1.Service
 // NewIngress makes an Ingress using the provided ns/name for the
 // object's namespace/name and backendName/backendPort as the name
 // and port of the backend Service.
-func NewIngress(name, ns, backendName string, backendPort int) *networkingv1beta1.Ingress {
-	return &networkingv1beta1.Ingress{
+func NewIngress(name, ns, backendName string, backendPort int32) *networkingv1.Ingress {
+	return &networkingv1.Ingress{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: ns,
 			Labels:    map[string]string{"app": name},
 		},
-		Spec: networkingv1beta1.IngressSpec{
-			Backend: &networkingv1beta1.IngressBackend{
-				ServiceName: backendName,
-				ServicePort: intstr.IntOrString{IntVal: int32(backendPort)},
+		Spec: networkingv1.IngressSpec{
+			DefaultBackend: &networkingv1.IngressBackend{
+				Service: &networkingv1.IngressServiceBackend{
+					Name: backendName,
+					Port: networkingv1.ServiceBackendPort{
```

Comment:
@sheetalsingala instead of changing the signature of `NewIngress`, can you set `backendPort` to an int32, i.e. `int32(backendPort)` in the function?

Reply:
I can do that. Thank you for the feedback